src/components/step-sequence/step-sequence-item/step-sequence-item.js

```diff
@@ -0,0 +1,35 @@
+import Icon from 'components/icon';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import './step-sequence-item.scss';
+
+const baseClass = 'carbon-step-sequence-item';
+const classes = state => `${baseClass} ${baseClass}--${state}`;
+
+const stepMarker = (state, indicator) => {
+  return state === 'complete' ? <Icon type='tick' /> : indicator;
+};
+
+const StepSequenceItem = ({ label, state, indicator }) => (
+  <li className={ classes(state) }>
+    <span className='carbon-step-sequence-item__label'>
```

Comment:
is this `span` actually required? looks like it may be superfluous

Reply:
This is required to bind the two elements together in order to keep them in line when the sequence switches to vertical,

Without it they were vertically oriented. I tried `flex` and `grid` to fix this but it became complex before working so this seemed a better solution